### PR TITLE
fix home page and add order by times

### PR DESCRIPTION
**Pull request by hapo-chinhnc**

### CODE REVIEW

**Review by hapo-lam-nv — COMMENTED** [review]
LGTM

**Review by Hapo-congbv — APPROVED** [review]
LGTM

**Review by hapo-hungnn97 — APPROVED** [review]
LGTM

File: resources/views/vendor/pagination/page_pagination.blade.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
+@if ($paginator->hasPages())
```
Comment (hapo-anhnhn) [inline resources/views/vendor/pagination/page_pagination.blade.php:1]: phải custom file panigate hả em

**Pull request merged**